runtime/doc/autocmd.txt

```diff
@@ -424,6 +424,7 @@ Name			triggered by ~
 
 |RemoteReply|		a reply from a server Vim was received
 
+|QuickFixCmdChanged|	after calling setqflist or setloclist function
 |QuickFixCmdPre|	before a quickfix command is run
 |QuickFixCmdPost|	after a quickfix command is run
```

Comment:
There should be a definition of the `*QuickFixCmdChanged*`  tag with some more details